File: application/core/src/main/java/br/com/beblue/bluerecords/core/repositorio/DiscoSpotifyClient.java
package br.com.beblue.bluerecords.core.repositorio;

import br.com.beblue.bluerecords.core.command.CadastraDiscoCommand;
import br.com.beblue.bluerecords.core.entitidade.Genero;

import java.util.List;

public interface DiscoSpotifyClient {
    List<CadastraDiscoCommand> buscarDiscos(Genero genero, Integer limit, Integer offset);
}

File: application/core/src/main/kotlin/br/com/beblue/bluerecords/core/repositorio/DiscoSpotifyClient.kt
package br.com.beblue.bluerecords.core.repositorio

import br.com.beblue.bluerecords.core.entity.Genero
import br.com.beblue.bluerecords.core.usecase.disco.command.CadastrarDiscoCommand

interface DiscoSpotifyClient {
    fun buscarDiscos(genero: Genero, limit:Int = 50, offset:Int = 0) : List<CadastrarDiscoCommand>
}